src-tauri/src/behavior/farming_behavior.rs

```diff
@@ -127,20 +129,38 @@ impl<'a> FarmingBehavior<'_> {
             }
         }
     }
+    fn debug_stats_bar(&mut self, config: &FarmingConfig, image: &ImageAnalyzer){
+        let fp  = image.detect_stats_bar(self.last_hp,"FP");
+        let fpp = fp.unwrap_or_default();
 
+        let mp  = image.detect_stats_bar(self.last_hp,"MP");
+        let mpp = mp.unwrap_or_default();
+
+        let hp  = image.detect_stats_bar(self.last_hp,"HP");
+        let hpp = hp.unwrap_or_default();
+
+        let exp  = image.detect_stats_bar(self.last_hp,"EXP");
+        let expp = exp.unwrap_or_default();
```

Comment:
I think for correct tracking you'd need to add `self.last_fp`, `self.last_mp` and `self.last_exp` fields to `FarmingBehavior`, so that you can pass the proper values. Otherwise tracking will be incorrect.